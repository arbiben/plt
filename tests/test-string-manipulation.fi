$this test file tests spliting a string by delimiter, converting it to upper case, and then concatanating it to another string$
$Not possible in microc.$
$Expected output: APRIL is the 4th month$


int main(){
str months?
str numbers?
str april?
str txt1?
str txt2?
str four?
str result?

txt1 = "IL is the "?
txt2 = "th month"?

months = "JAN, FEB, MAR, APR, MAY, JUN, JUL, AUG, SEP, OCT, NOV, DEC"
mumbers = "1,2,3,4,5,6,7,8,9,10,11,12"

april = getUp(split(months, ",", 3))?
four = split(numbers, ",", 3)?

result = concat(concat(april, txt1), concat(four,txt2))?

printstring(result)?

return 0 
}
