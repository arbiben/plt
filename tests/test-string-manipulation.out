 APRIL is the 4th month
